lib/sidekiq_fast_debounce.rb

```diff
@@ -80,15 +76,26 @@ module ClassMethods
     # @param delay [#to_f] how many seconds to delay to wait for other jobs
     # @param args [*Array]
     # @return [String] Sidekiq job id
+    def perform_debounce(delay, *args)
+      at = Time.now.to_f + delay.to_f
```

Comment:
no need for the first `.to_f`
```suggestion
      at = Time.now + delay.to_f
```

Reply:
Turns out you do need the `to_f`.
```
ArgumentError:  Job 'at' must be a Numeric timestamp: ...
```